chart/orkestra/values.yaml

```diff
@@ -115,6 +120,14 @@ plugins:
 # USE plugins.keptn to configure these values
 gitea:
   enabled: *keptnEnabled
+  url: *gitURL
```

Comment:
Question: why do these need to be specified at all? Can't we just dynamically generate these things? In general, I figured we would just need to take admin credentials at `gitea` install time